src/components/LanguageSelector/LanguageSelectorButton.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react'
+import classnames from 'classnames'
+
+type LanguageSelectorButtonProps = {
+  label: string
+  labelAttr?: string
```

Comment:
Do we still need the `labelAttr` prop? It seems that in order for this prop to be utilized, `LanguageSelectButton` would need to be used outside of the `LanguageSelector` component.

Reply:
it's still useful for the 2 language variant, where the language's text should be tagged. I've updated it a bit to reflect that